Add command example to create a capped collection (#2065)
doctrine/mongodb-odm: docs/en/reference/capped-collections.rst

    <?php

    $documentManager->getSchemaManager()->dropDocumentCollection('Category');

> If using Symfony in addition to Doctrine, use the following command to create the capped collection:

Reviewer: The console commands are available even if not using Symfony, but people need to add them to their console application themselves. I'd reword this:
```suggestion
Use the ``odd:schema:create`` command to create the database schema:
```

Reviewer 2: Watch out for typo in ``odd:schema:create`` in suggested changes ;)

Reviewer: Whoops. Thanks @Steveb-p, I've fixed it.

> .. code-block:: bash

Reviewer: Elsewhere, we use `console` instead of `bash`
```suggestion
.. code-block:: console
```

>     php app/console --env=<env> doctrine:mongodb:schema:create

Reviewer: This is how we show console examples elsewhere, as we don't assume people to use the Symfony Bundle:
```suggestion
    $ php mongodb.php odm:schema:create
```
